docs/design-document.md

```diff
@@ -64,76 +73,192 @@ Repository
 ```
 
 
-### Actors
+### Actors and Authentication
 
-In a Git repository, an "actor" is any user who makes changes to the repository.
-These changes can involve any part of the repository, such as modifying files,
-branches or tags. In the gittuf system, each actor is identified by a unique
-signing key that they use to sign their contributions. This means that when a
-policy allows an actor to make certain changes, it's actually allowing the
-person who has the specific signing key to make those changes. To maintain
-security, all actions made in the repository, such as adding or modifying files,
-are checked for authenticity. This is done by verifying the digital signature
-attached to the action, which must match the public key associated with the
-actor who is supposed to have made the change.
+In a Git repository, an "actor" is any party, human or bot, who makes changes to
+the repository. These changes can involve any part of the repository, such as
+modifying files, branches or tags. In gittuf, each actor is identified by a
+unique signing key that they use to cryptographically sign their contributions.
+gittuf uses cryptographic signatures to authenticate actors as these signatures
+can be verified by anyone who has the corresponding public key, fundamental to
+gittuf's mission to enable independent verification of repository actions. Note
+that gittuf does not use Git commit metadata (e.g., author email, committer
```

Comment:
Is it true we never use this information for anything?   What if it doesn't match what we expect for a user?   Would it be more accurate to position this as information we don't rely on?

Reply:
We don't use it at all at the moment, not even for whether it matches for a user. We don't have a "source of truth" for that information. We could infer in some cases if a person uses sigstore to sign by looking at the email + provider but there's no reason for the oidc identity used there to match the committer email.

I've updated the text to say "rely on" rather than "use".